### PR TITLE
Packetevent patch

### PR DESCRIPTION
**Pull request by papejik**

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/event/EventManager.kt [hunk 1]
```diff
@@ -65,7 +66,6 @@ object EventManager {
         if (!handlers.contains(hook)) {
             handlers.add(hook)
 
-            handlers.sortByDescending { it.priority }
```
Comment (1zun4secondary) [inline src/main/kotlin/net/ccbluex/liquidbounce/event/EventManager.kt]: Please add priority sorting back.

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleChestStealer.kt [hunk 2]
```diff
@@ -32,33 +32,35 @@ object ModuleChestStealer : Module("ChestStealer", Category.PLAYER) {
 
     val timer = Chronometer()
 
-    val repeatable = handler<GameRenderEvent> {
```
Comment (1zun4secondary) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleChestStealer.kt]: Revert these changes.

**Review by 1zun4secondary — CHANGES_REQUESTED** [review]

**Comment by 1zun4secondary** [comment]
Looks good now.

**Comment by papejik** [comment]
Sorry for the PR mess, I haven't quite figured it out yet

**Comment by papejik** [comment]
How do I PR for a specific commit?

**Comment by superblaubeere27** [comment]
You have to create a branch for each PR
